.github/workflows/ci.yml

```diff
@@ -13,7 +13,8 @@ jobs:
     strategy:
       matrix:
         rust: [stable, beta]
-    runs-on: ubuntu-latest
+        os: [ windows-latest, ubuntu-latest ]
```

Comment:
I think we need a separate job for Windows. The current job relies on quite a few bash scripts (lint.sh, test.sh etc)